packages/playground/blueprints/public/blueprint-schema.json

```diff
@@ -94,7 +94,8 @@
 							}
 						]
 					},
-					"description": "WordPress plugins to install and activate"
+					"description": "WordPress plugins to install and activate",
+					"deprecated": "This experimental option will change without warning.\nUse `steps` instead."
```

Comment:
This needs a rebase

Reply:
[1459d40](https://github.com/WordPress/wordpress-playground/pull/1538/commits/1459d40bb2245ff926839341752ea529584ba546)